flang/test/Driver/print-pipeline-passes.f90

```diff
@@ -0,0 +1,10 @@
+! Test that -print-pipeline-passes works in flang
+
+! RUN: %flang_fc1 -triple x86_64-unknown-linux-gnu -emit-llvm-bc -o /dev/null -mllvm -print-pipeline-passes -O0 %s 2>&1 | FileCheck %s
```

Comment:
Why does this need to specify the triple?

Reply:
That is a good point. I just modeled it off of the corresponding test in `clang`, but it should work just fine for any host.